pkg/controller/watch/resource_watcher_test.go

```diff
@@ -62,4 +62,5 @@ func TestEnsureResourcesAreWatched(t *testing.T) {
 		}
 		assert.Equal(t, expectedWatched, watcher.WatchedResources)
 	})
+	// TODO: add test for different kind of resources
```

Comment:
same as above, do we have ticket for this?

Reply:
Good point. I'll create a ticket